README.md

```diff
@@ -54,15 +54,17 @@ repositories {
  * Artifact:
 ```groovy
 dependencies {
-    compileOnly 'com.destroystokyo.paper:paper-api:1.16.5-R0.1-SNAPSHOT'
+    compileOnly 'io.papermc.paper:paper-api:1.17-R0.1-SNAPSHOT'
 }
 ```
 
 How To (Compiling Jar From Source)
 ------
-To compile Paper, you need JDK 8, maven, and an internet connection.
+To compile Paper, you need JDK 16 and an internet connection.
 
-Clone this repo, run `./paper jar` from *bash*, get files.
+Clone this repo, run `./gradlew applyPatches`, then `./gradlew reobfJar` from bash. You can find the compiled jar in the `Paper-Server/build/libs` directory.
```

Comment:
oh and well, bash isnt a requirement, so maybe `from your terminal` instead?

Reply:
![pufferfish](https://user-images.githubusercontent.com/28825609/122652704-6060c980-d140-11eb-81a1-93e84a2cc07a.png)